fili-core/src/test/groovy/com/yahoo/bard/webservice/web/endpoints/ErrorDataServletSpec.groovy

```diff
@@ -444,6 +445,43 @@ class ErrorDataServletSpec extends Specification {
         GroovyTestUtils.compareErrorPayload(r.readEntity(String.class), jsonFailure)
     }
 
+    def "dateTime must always be the first field in the sort list"() {
+        String message = DATE_TIME_SORT_VALUE_INVALID.format()
+
+        String jsonFailure =
+                """{"status":400,
+                    "statusName": "Bad Request",
+                    "reason":"com.yahoo.bard.webservice.web.BadApiRequestException",
+                    "description":"${message}",
+                    "druidQuery":null,
+                    "requestId": "SOME UUID"
+                   }
+                """
+
+        when:
+        Response r = jtb.getHarness().target("data/shapes/day/color")
+                .queryParam("metrics","height")
+                .queryParam("dateTime","2014-09-01%2F2014-09-10")
+                .queryParam("sort","height|ASC,dateTime|DESC")
+                .request().get()
+
+        then:
+        r.getStatus() == Response.Status.BAD_REQUEST.getStatusCode()
+        GroovyTestUtils.compareErrorPayload(r.readEntity(String.class), jsonFailure)
+    }
+
+    def "Successful execution if dateTime is first field in sort list"() {
+        when:
+        Response r = jtb.getHarness().target("data/shapes/day/color")
+                .queryParam("metrics","height")
+                .queryParam("dateTime","2014-09-01%2F2014-09-10")
+                .queryParam("sort","dateTime|DESC,height|ASC")
```

Comment:
I feel like this "success" test doesn't belong in the `ErrorDataServletSpec`, I feel like it belongs in a Spec that's focused on sorting. If there isn't already a place that's testing that, we should make one 😉